jupyter/datascience/ubi9-python-3.12/setup-kale.sh

```
@@ -0,0 +1,105 @@
+#!/bin/bash
+set -x
+
+# RHOAIENG-57465: Setup script for Kubeflow Kale JupyterLab extension
+# This script configures Kale to connect to KFP by reading Elyra runtime config
+
+# Read Elyra config and copy the relevant information to Kale config
+# Extract KFP configuration from Elyra runtime configs if available
+if [ "$(ls -A /opt/app-root/runtimes/ 2>/dev/null)" ]; then
+  # Get the first Elyra runtime config file
+  ELYRA_RUNTIME_CONFIG=$(ls /opt/app-root/runtimes/..data/*.json 2>/dev/null | head -1)
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)